taxi/migrations/0002_alter_manufacturer_options_and_more.py

```diff
@@ -6,17 +6,17 @@
 class Migration(migrations.Migration):
 
     dependencies = [
-        ('taxi', '0001_initial'),
+        ("taxi", "0001_initial"),
     ]
 
     operations = [
         migrations.AlterModelOptions(
-            name='manufacturer',
-            options={'ordering': ['name']},
+            name="manufacturer",
+            options={"ordering": ["name"]},
```

Comment:
The `ordering` option for the `Manufacturer` model is set to order by `name`. Ensure that this aligns with your application's requirements for displaying manufacturers.